src/controllers/property.js

```diff
@@ -20,9 +21,16 @@ class PropertyController{
                     };
                         
                 }
+        // const userResult = userArray.find(user => req.body.email === user.email);
```

Comment:
Good job @RachealN please see to it that you remove this commented out code.

Reply:
Alright.Thanks